syntax = "proto3";

package fixbug;

option cc_generic_services = true;
message LoginRequest
{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse
{
    int32 errno_code = 1;
    bytes errmsg = 2;
    bool success = 3;
}

service UserServiceRpc{
    rpc LoginService(LoginRequest) returns(LoginResponse);
}
